apps/wallet-mobile/src/features/Nfts/useCases/Nfts.tsx

```diff
@@ -40,6 +41,12 @@ export const Nfts = () => {
     return isSearching ? balances.nfts.filter(({info}) => byName(info)) : balances.nfts
   }, [balances.nfts, isSearching, search])
 
+  const {invalidate, isLoading} = usePortfolioImageInvalidate()
+  const onRefresh = React.useMemo(
```

Comment:
it'd invalidate all list, for now I'd say to add it to the `NFTDetails/MediaDetails` only, (I'm afraid that if the users starting pulling it too often we will end up reprocessing tons of images which a little bit expensive).

Please add it to the TokenList (details) inside the Portfolio too, so over there we can also invalidate the tokens (I dunno if it is using the MediaDetails) if it's just disregard.

Reply:
Same component.
I'll remove the mass invalidate from the gallery.